pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -310,6 +321,12 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *CAOptions) (
 		rootCertFile = ""
 	}
 
+	if customca.IsUsingCustomCAForWorkloadCertificate(s.environment.Mesh().GetCa()) {
```

Comment:
seems like below this we still set up signing key, etc. Do we still want that?

Reply:
Can you explain me in more detail?
I made a function to make it clearer to understand. 
This only support for Workload Certificates, For the DNS, Internal Istio Certificates are still using Self Signing Certificates. I got some struggle when implement full Certificates and I don't know why :( . Our customers need this feature, so I implemented for Workload Certificates before. 
I'll have future work for supporting full certificates for Custom CA